### DISCUSSION TITLE
Training mistral instruct - dataset and prompt format

### DISCUSSION THREAD

**Discussion by dinonovak** [axolotl-ai-cloud/axolotl#1093] [Q&A]
Hi,
I am trying to set up training for mistral instruct, can you please advise what dataset config to use.
I have dataset configured in instruct format https://huggingface.co/datasets/dnovak232/sql_create_context-v4-mssql-instruct

but how do I need to configure .yaml for it. I see this recommendation for instruct format below, but not sure as mistral uses <s>[INST] {instruction} [/INST] {answer} </s>  ?

datasets:
  - path: repo
    type:
      system_prompt: ""
      field_system: system
      format: "[INST] {instruction} [/INST]"
      no_input_format: "[INST] {instruction} [/INST]"

**Comment by gardner** [comment]
It looks like it matches the `alpaca` format described on the README.md

Have you tried something like this?

```
datasets:
  - path: dnovak232/sql_create_context-v4-mssql-instruct
    type: alpaca
```

**Comment by NanoCode012** [comment]
Yes! The one you said works. Alternatively, you can use the following
```
type: sharegpt
conversation: mistral
```
